bbcode-src/tags/subscript.js

```diff
@@ -0,0 +1,16 @@
+import { toNode } from "../utils/common";
+
+/**
+ * @file Adds subscript to bbcode
+ * @example [sub]content[/sub]
+ */
+
+const sub = (node) => {
+  return toNode(
+    "sub",
+    { style: "font-variant-position: sub;" },
```

Comment:
i thought `<sub>` and `<sup>` already have their own styles applied by the browser? No need for `font-variant-position`

Reply:
I didn't create them via the HTML tags, would that be preferred? I can change them if that's the case!

Reply:
I meant i didn't think the style parameter is necessary here.

Reply:
Alright! I've changed them to use the HTML tags!